packages/react-router/src/route.ts

```diff
@@ -12,45 +12,45 @@ import type {
   AnyPathParams,
   AnySchema,
   AnyValidator,
+  BeforeLoadContextParameter,
+  BuildLocationFn,
   Constrain,
   ConstrainLiteral,
+  RootRoute as CoreRootRoute,
+  Route as CoreRoute,
```

Comment:
Should we follow this and prefix all of these imports from `router-core` with `Core`? or is the DX without adding this prefix fine?

Reply:
tbh this was just a way for me to alias the Route type in core but keep the current Route class in this file